networkx/algorithms/flow/__init__.py

```diff
@@ -1,12 +1,6 @@
-from .maxflow import *
-from .mincost import *
-from .edmondskarp import *
-from .preflowpush import *
-from .shortestaugmentingpath import *
-from .capacityscaling import *
-from .networksimplex import *
-from .utils import build_flow_dict, build_residual_network
-
+# this import is used to combine the __all__ varibles.
```

Comment:
What's the point of lines 1–12?  If you just do the imports on lines 14–21, you don't need the effect of __all__, right?